mailarchive/mailarchive-tool/tool/src/webapp/vm/mailarchive/chef_mailbox-List.vm

```diff
@@ -59,7 +59,7 @@
 		#set ($sort-img = "sakai/sortdescending.gif")
 		#set ($sort-dir = "descending")
 	#end
-	#if($allMsgNumberInt > 0)
+	#if($!allMsgNumberInt && $allMsgNumberInt > 0)
```

Comment:
You don't need the `!`, all that does is cause velocity to output an empty string rather than the variable as this is in a test you can just do `($allMsgNumberInt && $allMsgNumberInt > 0)` which is a little easier tot read.